frontend/src/pages/groups/GroupList.tsx

```diff
@@ -0,0 +1,114 @@
+import { FunctionComponent, useMemo } from 'react'
+import { useSortBy, useTable } from 'react-table'
+import cx from 'classnames'
```

Comment:
nitpick (non-blocking): thoughts on calling this variable `cs` or `classNames` or somesuch? `cx` doesn't evoke "class names" for me.

Reply:
There's [a bit of history there](https://github.com/JedWatson/classnames/issues/157) 🤓 I am lazy and find `classNames` to be too long to type and a little difficult to distinguish from `className` at a glance (bad eyesight over here.) I wouldn't mind using `cn` or `cs`, but I've seen `cx` used in enough codebases that I think it's an established convention. @andyjac what do you think?

Reply:
Ah ok, sounds like I'm the odd one out then 😆, I can get used to `cx` if it's conventional

Reply:
Yeah, I've seen `cx` in other projects also, so I think this is pretty standard and would be friendly to other front-end folks as well.